### PR TITLE
Add ```us.kg```

### PR DESCRIPTION
**Pull request by EdwardLab**
Add us.kg DNS public suffixes
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

Organization Website: https://digitalplat.org

A global non-profit organization that supports open source and community development and explores innovative projects, I am the Founder of DigitalPlat.

We see that many people cannot build a website because of the expensive domain name. DigitalPlat is a non-profit organization (I am the Founder), and we are using US.KG to open a free domain name website. Anyone can get their own third-level domain (free subdomain) for projects or personal use that don't require a paid domain

The original owner of this domain requested deletion in #1741, and I took over us.kg to continue the mission of free subdomains

We're close to release and would like PSL inclusion before launch for the reasons listed below. Right now, during internal testing phase, there are only a few domains active.

Reason for PSL Inclusion
====
All Third Level Domains are completely independent and should be protected by browsers' security features. (e.g. to prevent setting cookies on the parent domain and hostname highlighting where supported)

We allow name server delegation of our Third-Level-Domains by NS record and we've learned that some DNS, hosting and proxy services validate domains using the PSL in order to prevent users from accidentally adding subdomains instead of their main domain.

We will not use wildcard TLS certificates on the parent domains nor will there be any content.

Subdomain owners cannot use DNS management platforms such as Cloudflare to manage their own subdomain DNS

DNS Verification via dig
=======

```
dig +short TXT _psl.us.kg
"https://github.com/publicsuffix/list/pull/1755"
```

Results of Syntax Checker (`make test`)
=========
```
13766: error: Found doublette/ambiguity (previous line was 13608): 'us.kg'
make: *** [Makefile:12: test-syntax] Error 1
```
The original owner of this domain requested deletion in #1741, and I took over us.kg to continue the mission of free subdomains

### CODE REVIEW

**Comment by groundcat** [comment]
While the DNS Verification via dig is still valid, the description of the organization in the PR does not match the information presented on the registry website? According to the `https://nic.us.kg/about/` page [1], the us.kg registry is managed by SanWardTech Global, a California-based company, and the domain costs $2 per year [2]. However, you indicated in your PR that the domain has been acquired by DigitalPlat, a non-profit organization using US.KG to launch a free domain website.
Meanwhile, note that 2 security vendors flagged this URL as malicious [3] @dnsguru 

[1] https://archive.is/4yM9M
[2] https://archive.is/7GMbR
[3] https://www.virustotal.com/gui/url/f9082214f400071eb313d0cf8a0fd14462b68aff7d78f64252c64143d1003140/detection

**Comment by xingyujie** [comment]
> While the DNS Verification via dig is still valid, the description of the organization in the PR does not match the information presented on the registry website? According to the `https://nic.us.kg/about/` page [1], the us.kg registry is managed by SanWardTech Global, a California-based company, and the domain costs $2 per year [2]. However, you indicated in your PR that the domain has been acquired by DigitalPlat, a non-profit organization using US.KG to launch a free domain website. Meanwhile, note that 2 security vendors flagged this URL as malicious [3] @dnsguru - might deserve a security review
> 
> [1] https://archive.is/4yM9M [2] https://archive.is/7GMbR [3] https://www.virustotal.com/gui/url/f9082214f400071eb313d0cf8a0fd14462b68aff7d78f64252c64143d1003140/detection

Thank you for your response!

Due to developments since the initial writing of this PR, there have been some updates to consider. While US.KG is committed to establishing a free domain name, the DigitalPlat Foundation is currently facing financial constraints, and unfortunately, we have received only a very small amount of donations. Despite our organization's collaborative efforts to contribute towards the server and domain name expenses, we are finding it challenging. As a result, we have decided to set the initial price at $2. However, the ultimate goal is to transition to a model where the domain name is provided for free once we gather sufficient funds to cover the costs associated with the "invested" domain name.

For verification purposes, you can reach me via email at contact@digitalplat.org.

Best regards,
Edward Hsing
Founder of DigitalPlat

**Review by dnsguru — CHANGES_REQUESTED** [review]
Please move this higher in the private section so that the entry is listed where // DigitalPlat would fit sorted a-z.

This is being considered to process with #1741 which would remove it from the PSL, so we're watching for this change and will process your request once appropriately revised to be correctly sorted, if it also has the DNS _PSL TXT record and registration period requirements met.

File: public_suffix_list.dat [hunk 1]
```diff
@@ -11228,6 +11228,10 @@ ondigitalocean.app
 // Submitted by Robin H. Johnson <psl-maintainers@digitalocean.com>
 *.digitaloceanspaces.com
 
+// DigitalPlat : https://www.digitalplat.org/
+// Submitted by Edward Hsing <xingyujie@digitalplat.org>
```
Comment (simon-friedberger) [inline public_suffix_list.dat:11232]: Please provide a non-personal e-mail.

**Review by simon-friedberger — CHANGES_REQUESTED** [review]

File: public_suffix_list.dat (hunk 1 above)
Comment (EdwardLab) [reply public_suffix_list.dat]: Updated

**Comment by simon-friedberger** [comment]
@EdwardLab Is this obsolete?

**Comment by EdwardLab** [comment]
Nope, I'll renewal domain for two years these days, thank you

**Comment by simon-friedberger** [comment]
Merging now since I have also merged the removal.

**Comment by pencilnav** [comment]
@EdwardLab Upon checking, an entry associated with this PR #1755 (us.kg) is still being flagged by Virustotal.

Please contact respective security vendors to address this issue.

https://www.virustotal.com/gui/domain/xx.kg

Chong Lua Dao: Malicious
alphaMountain.ai: Suspicious
Gridinsoft: Suspicious 

{2D9E87F0-A2D6-4A79-93F5-D43D0D8B96F8}

**Pull request merged**